beta/src/components/MDX/ExpandableExample.tsx

```diff
@@ -22,17 +22,29 @@ function ExpandableExample({
   excerpt,
   type,
 }: ExpandableExampleProps) {
+  const ref = React.useRef<null | HTMLDetailsElement>(null);
   const [isExpanded, setIsExpanded] = React.useState(false);
   const isDeepDive = type === 'DeepDive';
   const isExample = type === 'Example';
 
   return (
-    <div
+    <details
+      open={isExpanded}
+      onToggle={(e: any) => {
+        setIsExpanded(e.target!.open);
```

Comment:
(nit: shouldn't this be currentTarget for the hypothetical case of nested details tags?)

Reply:
hmm i guess so

Reply:
sophienits